functions/Invoke-DbaDatabaseShrink.ps1

```diff
@@ -117,13 +127,20 @@ Shrinks all databases on SQL2012 (not ideal for production)
 		[int]$PercentFreeSpace = 0,
 		[ValidateSet('Default', 'EmptyFile', 'NoTruncate', 'TruncateOnly')]
 		[string]$ShrinkMethod = "Default",
+		[ValidateSet('All', 'Data', 'Log')]
+		[string]$FileType = "All",
```

Comment:
Need to add `LogsOnly` as an alias to this parameter if we are removing it. This prevents commands from failing and does not require us to keep the parameter on the list and help for it.

Reply:
@wsmelton wouldn't that be a bit confusing since LogsOnly is a switch, and FileType is a string?

Reply:
ahh yeah, good point, Shawn. It's our deprecation feature, @dgmyrs where it doesnt throw an error and it just says that its deprecated. brb ill put a lil patch

Reply:
oh, @dgmyrs already has most of it :D agreed with Doug in his approach beecause of the switch vs string and he takes care of it within the code.

Reply:
It won't matter in usage @dgmyrs because we just want the name alias so it "points" to the new parameter, the type is irrelevant in this case.

Reply:
Using the alias method allows us to go ahead and update the help documentation now. It is one less thing to deal with when we have to go through all the commands and remove the deprecated messages.